app/Services/CourseService.php

```diff
@@ -0,0 +1,17 @@
+<?php 
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Repositories\CourseRepository;
+
+class CourseService extends CrudService{
```

Comment:
after Crudservice there would be a line break.

But I would suggest using a formatter to format PSR-12 format. You can install some VS code extension to do that.

alt+shift+f by default maybe in windows.